sites/public/src/pages/applications/contact/address.tsx

```diff
@@ -72,6 +72,7 @@ const ApplicationAddress = () => {
     mergeDeep(application, data)
     if (newAddressSelected && foundAddress.newAddress) {
       application.applicant.applicantAddress.street = foundAddress.newAddress.street
+      application.applicant.applicantAddress.street2 = foundAddress.newAddress.street2
```

Comment:
This was the bug causing the street2 on the primary applicant to not come through